docs/content/v2024.2/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -339,21 +343,13 @@ The `help`, `license`, and `preflight` commands can be run in either context.
 If you don't use the correct execution path, yba-ctl fails with an error:
 
 ```sh
-sudo ./yba-ctl createBackup ~/backup.tgz
+./yba-ctl createBackup ~/backup.tgz
 ```
 
 ```output
 FATAL[2023-04-25T00:14:57Z] createBackup must be run from the installed yba-ctl
 ```
 
-## Non-sudo installation
-
-YBA Installer also supports a non-sudo installation, where sudo access is not required for any step of the installation. Note that this is not recommended for production use cases.
```

Comment:
lets keep this block, but we can remove the statement "Note that this is not recommended for production use cases."